tests/e2e/kyma_metrics_test.go

```diff
@@ -72,17 +74,20 @@ var _ = Describe("Manage Module Metrics", Ordered, func() {
 		})
 
 		It("When Kyma Module is disabled", func() {
+			manifestInCluster, err := GetManifest(ctx, controlPlaneClient, kyma.GetName(), kyma.GetNamespace(),
+				module.Name)
+			Expect(err).Should(Succeed())
 			Eventually(DisableModule).
```

Comment:
what you suggest is in `module_deletion_test.go`, this one is for testing metrics, and why I have to modify this test is because of the reason [here](https://github.com/kyma-project/lifecycle-manager/pull/1336#discussion_r1515882254)